ArcGISRuntimeSDKQt_QMLSamples/EditData/EditFeaturesWithFeatureLinkedAnnotation/deployment.pri

```diff
@@ -0,0 +1,43 @@
+#-------------------------------------------------
+# Copyright 2020 Esri.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#-------------------------------------------------
+
+
+android-no-sdk {
+    target.path = /data/user/qt
+    export(target.path)
+    INSTALLS += target
+} else:android {
+    x86 {
+        target.path = /libs/x86
+    } else: armeabi-v7a {
+        target.path = /libs/armeabi-v7a
+    } else {
+        target.path = /libs/armeabi
+    }
+    export(target.path)
+    INSTALLS += target
```

Comment:
Is this section still needed for Android with Qt 5.15? `/libs/armeabi` doesn't match any of the ABI targets.

Reply:
@JamesMBallard it definitely is not needed any more. This has come up before and I though that I just had an outdated version of the templates but it turns out this is still included in the sample templates for qml. I'll create an issue to get that cleaned up so this doesn't come up again.